src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,141 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import React from 'react';
+
+import { ComponentPropsWithoutRef } from 'react';
+import { MdArrowForwardIos } from 'react-icons/md';
+
+export interface PaginationProps extends ComponentPropsWithoutRef<'div'> {
+  pages: number;
+  currentPage: number;
+  customClass?: string;
+  onPageChange: (newPage: number) => void;
+}
+
+export function Pagination(props: PaginationProps) {
+  const { pages, currentPage, customClass, onPageChange } = props;
+  const pageClass = 'px-3 py-1 rounded-lg cursor-pointer';
+  const currentpageClass = `${pageClass} bg-f-primary text-f-secondary`;
+  const renderPageNumbers = (): JSX.Element[] => {
```

Comment:
```suggestion
  function renderPageNumbers() {
```
Can you move this function into a new file ?
And separate his content to multiple explicit function ?

Too complicated to understand.

Reply:
done